packages/shell-api/src/aggregate-or-find-cursor.ts

```diff
@@ -44,6 +44,7 @@ export abstract class AggregateOrFindCursor<
     if (verbosity !== undefined) {
       verbosity = validateExplainableVerbosity(verbosity);
     }
```

Comment:
Doesn’t this conditional explicitly skip applying this logic in the default case?

Reply:
You are right, but I was testing now the behaviour manually (and have some tests) and it seems to be fixed. I think I'm looking it with the wrong perspective.

Thanks for noticing it!